genesis-tools/keypair-generator/src/main.rs

```diff
@@ -60,30 +66,31 @@ fn main() {
                 println!();
 
                 pks.push(key.public_key());
-                if i == 0 && generate_config {
-                    let account_id = account_id
-                        .expect("Account id must be specified if --generate-config is used");
-                    let signer = InMemorySigner::from_secret_key(account_id.to_string(), key);
-                    let mut path = home_dir.to_path_buf();
-                    path.push(near::config::VALIDATOR_KEY_FILE);
-                    signer.write_to_file(path.as_path());
-                }
             }
             let pks: Vec<_> = pks.into_iter().map(|pk| format!("{}", pk)).collect();
             println!("List of public keys:");
             println!("{}", pks.join(","));
         }
-        ("node-key", Some(_args)) => {
+        ("validator-key", Some(_)) => {
```

Comment:
As discussed before I suggest we remove subcommands because we generate the same key format anyway. Then add two arguments `--save-validator-key` which saves the first key into the validator key file and  `--save-node-key` which saves the first key into the node key file.

Reply:
Do you want this tool to generate key pairs for accounts?

Reply:
Yes

Reply:
Then why don't we use subcommand? It seems clearer to me that way.

Reply:
Then we need to move --home and --generate-config into `validator-key` and `node-key` subcommands, because these args do not make sense for `signer-keys`.

Reply:
Why? I suppose people do want to write the generated key to some file.

Reply:
In the near future, I agree, we would need to write these keys to a file instead of printing them out. Ideally, that file should be encrypted. Currently, this file would be useless, since it does not have a format that other tools can understand, e.g. near-shell uses a different format AFAIU. Maybe this is something that you could verify.